### PR TITLE
Add an option to DaqTask to print header and payload

### PR DESCRIPTION
**Pull request by Barthelemy**
Just because it is handy and comes for almost free. Will be useless once we have a Data Inspector / Data Dump.

### CODE REVIEW

File: Modules/Daq/src/DaqTask.cxx [hunk 1]
```diff
@@ -111,11 +90,43 @@ void DaqTask::monitorData(o2::framework::ProcessingContext& ctx)
   // in a loop
   uint32_t totalPayloadSize = 0;
   for (auto&& input : ctx.inputs()) {
```
[the remark this answers is not in the document]
Comment (Barthelemy) [reply Modules/Daq/src/DaqTask.cxx:92]: correct

File: Modules/Daq/src/DaqTask.cxx [hunk 2]
```diff
@@ -111,11 +90,43 @@ void DaqTask::monitorData(o2::framework::ProcessingContext& ctx)
...
+      if(mCustomParameters.count("printPayload") > 0) {
+        std::vector<std::string> representation;
+        if(mCustomParameters["printPayload"] == "hex"){
+          representation = getHexRepresentation((unsigned char*)payload, header->payloadSize);
+        } else if (mCustomParameters["printPayload"] == "bin") {
+          representation = getBinRepresentation((unsigned char*)payload, header->payloadSize);
+        }
+
+        for(size_t i = 0 ; i < representation.size() ; ) {
+          cout << std::setw(4) << i << " : ";
```
[the remark this answers is not in the document]
Comment (Barthelemy) [reply Modules/Daq/src/DaqTask.cxx:115]: I hope ILOG accepts setw, let's see.

File: Framework/readout.json [hunk 3]
```diff
@@ -34,7 +34,11 @@
           "type": "dataSamplingPolicy",
           "name": "readout"
         },
-        "location": "remote"
+        "location": "remote",
+        "taskParameters": {
+          "printHeaders": "0",               "": "set to 1 to print all headers, it will slow down the QC",
```
[the remark this answers is not in the document]
Comment (Barthelemy) [reply Framework/readout.json:39]: yes

File: Modules/Daq/src/DaqTask.cxx (hunk 1 above)
[the remark this answers is not in the document]
Comment (Barthelemy) [reply Modules/Daq/src/DaqTask.cxx:92]: https://alice.its.cern.ch/jira/browse/QC-412

**Comment by Barthelemy** [comment]
ok, all done

**Review by knopers8 — APPROVED** [review]

**Pull request merged**